infrastructures/infrastructure-vmware/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/VMWareInfrastructure.java

```diff
@@ -99,6 +99,9 @@
     @Configurable(description = "Command used to download the worker jar")
     protected String downloadCommand = generateDefaultDownloadCommand();
 
+    @Configurable(description = "Optional list of MAC addresses separated by comma ',' to assign on new cloned VMs (1 per instance)")
```

Comment:
should we mention here that the number of mac addresses need to match with the number of instances?